src/components/Sections/SectionNovelty/index.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <section class="novelty">
+        <div class="novelty__box container">
+            <img src="../../../assets/new-coffee.png" alt="new-coffee picture"
+            class="novelty__picture">
+            <div class="novelty__describe">
+                <p class="novelty__before-title">What’s new?</p>
```

Comment:
According to design, it seems like a link - so I would replace **p** with **a** tag and change a class to novelty__extra-info